src/styles/constants/_typography.scss

```diff
@@ -1 +1,5 @@
 $bdl-line-height: 20px;
+
+// font sizes
+$bdl-font-size: 13px;
+$bdl-font-size--dejablue: 14px;
```

Comment:
fontSize, likewise camel casing other places too

Reply:
None of the other variables are like that

Reply:
I'm cool w/ whatever we go with, lmk!